src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+:root {
+  --color-font: #000;
+  --color-active: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", "Times New Roman", sans-serif;
+  color: var(--color-font);
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto;
```

this comment means delete only margin: auto
this style really redundant in this case
```suggestion
.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
```